src/server/frontend_wayland/window_wl_surface_role.cpp

```diff
@@ -364,6 +364,9 @@ void mf::WindowWlSurfaceRole::commit(WlSurfaceState const& state)
 
         if (pending_changes)
         {
+            auto content_size = scene_surface->content_size();
```

Comment:
I'm not going to say I'm happy with this, but both `committed_size` and `pending_size` are (surprisingly?) filled in by the time we're here…